src/adapters/adapter.ts

```diff
@@ -71,5 +72,22 @@ export abstract class Adapter {
    */
   public addModel(model: typeof Model): void {
     model.adapter = this;
+    this.models.push(model);
+  }
+
+  /**
+   * Returns an array containing all classes of the Models registered with 'addModel'.
+   */
+  public getAllModels(): Array<typeof Model> {
+    return this.models;
+  }
+  
+  /**
+   * Truncates all registered model tables with 'Model.truncate'.
+   */
+  public truncateAllModels() {
+    this.models.forEach(async (model) => {
```

Comment:
Firstly, this method needs to return a promise, so that we can wait until the tables are truncated before we do something else.

Secondly, I don't know about you, but I think it's `truncateAll` already makes sense for a method name.